tests/base/managers/entity.py

```diff
@@ -591,3 +591,56 @@ def test_mgr_entity_search_translation(entity_test_search):
     assert search('first translation') == [entities[0]]
     assert search('second translation') == [entities[1]]
     assert search('third translation') == [entities[2]]
+
+
+@pytest.mark.django_db
+def test_lookup_collation(resource0, locale0, translation_factory):
+    """
+    Filter translations according to collation.
+    """
+    entity = Entity.objects.create(
+        resource=resource0,
+        string="string")
+
+    batch_kwargs = sum(
+        [[dict(
+            locale=locale0,
+            entity=entity,
+            string='this is string')],
+         [dict(
+            locale=locale0,
+            entity=entity,
+            string='this is STRİNG')],
+         [dict(
+            locale=locale0,
+            entity=entity,
+            string='this is Strıng')],
+         [dict(
+            locale=locale0,
+            entity=entity,
+            string='this is StrInG')],
+         [dict(
+            locale=locale0,
+            entity=entity,
+            string='this is sTriNg')]],
+        [])
+    translations = translation_factory(
+        batch_kwargs=batch_kwargs)
+    assert (
+        set( Translation.objects.filter(
+            string__icontains_collate=(u'string', 'tr_tr')))
+        == {translations[0], translations[1], translations[4],})
```

Comment:
would you mind using `set()` rather than `{}` for sets as itss easy to confuse with `dicts`

Reply:
you could also do this...

```
batch_kwargs = [dict(entity=entity, locale=locale0, string=s) for s in [u'string 1', u'string 2']]
```